apps/dashboard/src/components/engine/configuration/webhooks-table.tsx

```diff
@@ -58,10 +58,11 @@ const columns = [
     header: "Secret",
     cell: (cell) => {
       return (
-        <AddressCopyButton
-          address={cell.getValue() || ""}
-          title="secret"
-          size="xs"
+        <CopyTextButton
```

Comment:
`AddressCopyButton` trims the text shown on the button - which you have to do manually on `CopyTextButton` 

production: 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/b3757fa9-75f3-4ee2-8cf1-ed1725380a90.png)

This PR: 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/0313ba10-3d93-4383-8212-adecd703e9f0.png)


please make sure the `textToShow` is trimmed properly on all changes done in this PR

Reply:
oh, ty ser